be/src/exec/json_parser.h

```diff
@@ -244,6 +244,6 @@ class ExpandedJsonArrayParserWithRoot : public JsonArrayParser {
     bool _curr_ready = false;
 };
 
-std::string format_json_parse_error_msg(const std::string& raw_error_msg);
+Status json_parse_error(const std::string& error_msg);
```

Comment:
explicit the name to `status_from_json_parse_error()`

Reply:
fixed